src/index.js

```diff
@@ -12,30 +12,51 @@ import BookingRepo from './Booking-repo';
 import TodayBookingRepo from './TodayBooking-repo';
 import UserBookingRepo from './UserBooking-repo';
 import Manager from './Manager';
+// import loginUser from './loginUser'
 import deleteBooking from './DeleteBooking';
 import domUpdates from './DomUpdates';
 import fetchAllData from './FetchAllData';
 import postNewBooking from './PostNewBooking';
 
+let user, manager, customer;
 window.addEventListener('load', onLoad);
 window.addEventListener('click', clickHandler);
 
-function onLoad() {}
+function onLoad() {
+  user = new User();
+}
 function clickHandler(event) {
   if (event.target.classList.contains('login-button')) {
     event.preventDefault();
     loginUser();
+    console.log(manager)
+    console.log(customer)
   }
 }
 
 function loginUser() {
-  console.log('ayyyyyyy')
+  const userName = document.querySelector('.username-input').value
+  const password = document.querySelector('.password-input').value
+  console.log(password)
+  login(userName, password);
 }
 
-function validatePassword() {
-
+function login(userName, password) {
+  if (userName === 'manager' && password === 'overlook2020') {
+    loginAsManager()
+  } else if (userName.slice(0, 8) === 'customer' && password === 'overlook2020') {
```

Comment:
Nice work with this logic!! How do you plan on incorporating the <id> portion of the login?

Reply:
This is how I'm going to decide if it's a manager or user logging in. So this part decides which display to make.